about.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Formula 1 Get Together Conference</title>
+    <link href='https://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="about.css">
+</head>
+<body>
+    <header class="hamburger-logo">
+
+        <div class="header-container">
+            <div><a href="#"><img src="images/mobile_menu1.svg" class="ham1"></a></div>
+            <nav>
+                <img src="images/F1_logo.png" class="logo">
+            </nav>
+        </div>
+    </header>
+
+    <nav class="menu" id="menu-pop">
+        <img src="images/Icon-close.svg" alt="" class="close-button">
+        <ul class="nav-content">
+            <li><a href="about.html">About</a></li>
+            <li><a href="#program" id="nav-grey">Program</a></li>
+            <li><a href="#join" id="nav-grey">Join</a></li>
+            <li><a href="#sponsor" id="nav-grey">Sponsor</a></li>
+            <li><a href="#news" id="nav-grey">News</a></li>
+        </ul>
+    </nav>
+
+    <section>
+        <div class="about-front">
+            <div class="about-intro">
+                <p id="about-hello">"Hello!,Sharing World"</p>
+                <div>
+                    <h1 class="about-title-intro">FORMULA 1 FANS GLOBAL SUMMIT 2023</h1>
+                </div>
+            </div>
+
+            <div class="global-intro">
+                <p>The Global Summit brings together the community of  
+                experts,academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+                This year, we hope to expand our invitation list including organizations and individuals.</p>
+            </div>
+            <div class="about-contact">
+                <p>Please contact us per Email for nay further questions about CC Global Summit 2015!</p>
+                <p id="about-email">ccsummit2015@cckorea.org</p>
+            </div>
+
+            <div class="about-crts">
+                <p>Creative Commons Global Summit 2015 Logo</p>
+                <hr class="about-hr">
+                <p class="att-dates">The logo of CC Global Summit 2015 was decided through the 
+                    logo competition from 8, June to 7, July.</p>
+
+                <hr class="about-sm-line">
+                <img src="images/about_logo 1.svg" class="about-logo">
```

Comment:
[REQUIRED] Kindly ensure that your design matches the template as instructed below:
![image](https://github.com/MarkYegon7/Project-Capstone/assets/62938735/ffc58280-7dfd-4ec9-b374-2d43a6f895b3)

Reply:
Fixed it accordingly